card/lib/card/simplecov_helper.rb

```diff
@@ -1,3 +1,6 @@
+def is_in_mod? file,mod_path
+  file.src.join("") =~ /below pulled from #{Rails.root}\/mod/#{mod_path}\//
+end
```

Comment:
What is file.src ?  Is file other than some standard File or related class?  It certainly isn't very clear what this is about.